modules/stochastic_tools/doc/content/source/reporters/ActiveLearningGPDecision.md

```diff
@@ -0,0 +1,83 @@
+# Active learning Gaussian Process Decision
```

Comment:
```suggestion
# ActiveLearningGPDecision
```

The reason behind this is that the user will see this line in the source index on the website.